daemon/rcutil.cpp

```diff
@@ -55,26 +122,31 @@ main(int argc, char* argv[])
 
   options.add_options()
       ("v,verbose", "Verbose", cxxopts::value<bool>())
-      ("c,colour", "colour output", cxxopts::value<bool>()->default_value("true"))
       ("h,help", "help", cxxopts::value<bool>())
-      ("j,json", "output in json", cxxopts::value<bool>())
```

Comment:
based on feedback from @jagerman, the non-json output isn't that useful, so make it the default.